app/models/user.rb

```diff
@@ -34,7 +34,7 @@
 #
 # Indexes
 #
-#  index_users_on_email                  (email)
+#  index_users_on_email                  (email) UNIQUE
```

Comment:
I'm pretty sure this SHOULD be here -- i keep having to remove it from my PRs when it gets generated, but it is in fact a unique index ?